model/game.rb

```diff
@@ -0,0 +1,34 @@
+require_relative './item'
+require 'date'
+require 'pry'
+
+class Game < Item
+  attr_reader :id
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date, id = Random.rand(1..999_999), archived = false)
+    super(publish_date, id, archived: archived)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    today_time = Time.now
+    today_date = Date.new(today_time.year, today_time.month, today_time.day)
+    (today_date - Date.parse(@publish_date)) > 36_525 && (today_date - Date.parse(@last_played_at)) > 730
+  end
```

Comment:
- [ ] for this method you should use the parent method instead of checking it again manually this would make it look something like:
```
super && (today_date - Date.parse(@last_played_at)) > 730
```

Reply:
add method to calcultate the years passed since current time:

´´´´
  def years_since_today(date1)
    today_time = Time.now
    today_date = Date.new(today_time.year, today_time.month, today_time.day)
    mydate1 = Date.parse(date1)
    ((today_date - mydate1) / 365.25).to_i
  end
´´´´